VIM_CheatSheet.md

```diff
@@ -3,6 +3,12 @@
 	#Insert mode
 		i
 
+    #Insert after cursor
+        a
+
+    #Create blank line below & insert
+        o
+
```

Comment:
Yo bro, small nitpick. Can you please make sure they are the correct spaces like the others please. 
Thanks